pootle_pytest/factories.py

```diff
@@ -41,3 +45,112 @@ class AgreementFactory(factory.django.DjangoModelFactory):
 
     class Meta(object):
         model = 'staticpages.Agreement'
+
+
+class DirectoryFactory(factory.django.DjangoModelFactory):
+
+    @factory.lazy_attribute
+    def pootle_path(self):
+        if self.parent is None:
+            return "/"
+        return (
+            "%s/%s"
+            % (self.parent.pootle_path.rstrip("/"),
+               self.name))
+
+    class Meta:
```

Comment:
@julen can you provide a link - i have never seen Meta inheriting from object - but may well be wrong